### ISSUE TITLE
[Frotend] Improvement in the Projects' cards 

### ISSUE THREAD

**Issue by filipe-varela** [HackerSchool/hs_app#5]
From a page where it should show the current projects being in development, it has cards for each one of latter showing the name and HS's members photos in avatar circles. Although this is great as basis, there still some improvements to be made.

For starters, the cards aren't showing exactly the current projects - it uses a hardcoded list in the code. To fix this, it should be used the HSInventory database - there is a table for the current projects in it - and retrieve the following information:
- [ ] Project's name
- [ ] The photos of HS's members in the same project

After this, the page should reorganized them in a way that should be responsive. I. e., there should be a list of cards where each of them are stacked one after another in a sense of not creating blank space in scenarios of a bigger card is besides a smaller one.

![projects](https://user-images.githubusercontent.com/43725698/184535681-19db17ec-242d-4e0c-87e6-6e578eb0e6c9.png)

**Comment by filipe-varela** [comment]
In continuation of this issue, it's worth to add the following requests so the experience in these cards should be complete instead being broken midway through. For that reason, the following information should be also retrieved for the project's page - a page that appears when the user presses on of the cards:
- [ ] The name of each member (possible it'll need an id for the member, so the project's page could create its own personal page)
- [ ] (If possible) Project's description
- [ ] (If possible) Project's date
- [ ] (If possible) Skills used

If possible, some of this information should also be used to build or the enable the building of the user's page, which can be accessed from the same project's page. Additionally, some of the information may not be stored in the HSInventory and that same information is marked with the *(if possible)*. In case that there is no information, that section is hidden so it could dealt in future issues or iterations of this project.

In case this issue is becoming more complex that apparent to be, it's recommended to create a new issue issuing the problems or tasks the complement this one.

In summary, the following tasks should be made in this issue for a better improvement of the Project's card/section behaviour:
- The cards should have the current and update information of the HS's projects
- They should be responsive in terms of size
- The page should retrieve information necessary to built the project's page
- Afterwords, the page should retrieve information to build or enabling the building of the HS's member's page

Some examples:
![project](https://user-images.githubusercontent.com/43725698/184535979-3aaa640d-d3d0-4b18-971b-e91ed1ee1efe.png)
![member](https://user-images.githubusercontent.com/43725698/184535982-37ad7ea9-e407-4aef-8957-d4a0793aa9c4.png)

**Comment by filipe-varela** [comment]
In addition of the previous tasks, it's necessary to make the font a bit more adaptive to the screen's user, since the current font is somewhat small for what it is expected, namely in the normal text after each section.
